crates/build/re_types_builder/src/codegen/rust/reflection.rs

```diff
@@ -170,21 +170,26 @@ fn generate_archetype_reflection(reporter: &Reporter, objects: &Objects) -> Toke
         let fqname = &obj.fqname;
         let quoted_name = quote!( ArchetypeName::new(#fqname) );
         let display_name = re_case::to_human_case(&obj.name);
-        let docstring_md = doc_as_lines(
-            reporter,
-            objects,
-            &obj.virtpath,
-            &obj.fqname,
-            &obj.docs,
-            Target::WebDocsMarkdown,
-        )
-        .join("\n");
+
+        if false {
+            // We currently skip the docstring for the archetype itself,
+            // because it is very loong and has mebedded examples etc.
+            // We also never use it.
```

Comment:
I'm fairly sure that's wrong. afaik we use doc strings of archetypes in view properties' tooltips. View properties are the reason generate reflection was introduced in the first place

... also this stretches what this PR was about terribly far 🤨

Reply:
If we were using them, then removing the field would have yielded a compilation error.